src/components/settings/Settings.h

```diff
@@ -50,6 +50,17 @@ namespace Pinetime {
         int colorIndex = 0;
       };
 
+      enum class HeartRateBackgroundMeasurementInterval : uint8_t {
+        Off,
+        Continuous,
+        TenSeconds,
+        ThirtySeconds,
+        OneMinute,
+        FiveMinutes,
+        TenMinutes,
+        ThirtyMinutes,
+      };
```

Comment:
I think that this is the wrong data structure for what you want to do. Maybe just use a cost array of integers representing seconds? Or at least set the enums values to that? 
In another file there is a function that maps the enums values to times. That all seems too complicated to me.

Reply:
I tried an array of integers before, but there was a problem when storing something longer than a `uint8_t` in the settings file. Therefore I opted to do it like this.

Reply:
Weird, but I guess it's alright then 😄